checkpointctl.go

```diff
@@ -148,13 +159,23 @@ func inspect(cmd *cobra.Command, args []string) error {
 		)
 	}
 
-	if psTree {
+	if psTree || psTreeEnv {
```

Comment:
I feel instead of this, we should move the if block for `psTreeEnv` up and set `psTree = true` in that, like how we do with `files`

Reply:
True, thank you @snprajwal